homework/index.js

```diff
@@ -29,19 +31,146 @@
     });
     return elem;
   }
+  /* cSpell:disable */
+  // Creating the repository-details part
+  function createHyfRepoDetails(leftDiv, hyfRepo) {
+    const table = createAndAppend('table', leftDiv, { class: 'table' });
+    const tBody = createAndAppend('tbody', table);
+    const details = ['Repository: ', 'Description: ', 'Forks: ', 'Updated: '];
+    details.forEach(detail => {
+      const tr = createAndAppend('tr', tBody);
+      createAndAppend('td', tr, {
+        text: detail,
+        class: 'label',
+      });
+      createAndAppend('td', tr);
+    });
+    const secondTd = document.getElementsByTagName('td')[1];
+    createAndAppend('a', secondTd, {
+      href: hyfRepo.html_url,
+      target: '_blank',
+      text: hyfRepo.name,
+      class: 'repo-name',
+    });
+    if (hyfRepo.description === null) {
+      document.getElementsByTagName('tr')[1].setAttribute('class', 'hide-description');
+    } else {
+      document.getElementsByTagName('td')[3].innerText = hyfRepo.description;
+    }
+    document.getElementsByTagName('td')[5].innerText = hyfRepo.forks;
+    document.getElementsByTagName('td')[7].innerText = new Date(
+      hyfRepo.updated_at,
+    ).toLocaleString();
+  }
```

Comment:
Consider Paul's comment https://github.com/HackYourFuture/JavaScript3/pull/206#discussion_r278256002
And instead of referencing the element you want to update by its tag name and order, it is better to give it an ID and use `document.getElementById`

Reply:
Thanks... It's now updated. 👍